templates/base.html

```diff
@@ -5,20 +5,16 @@
         <meta http-equiv="X-UA-Compatible" content="IE=edge">
         <meta name="viewport" content="width=device-width, initial-scale=1">
         <!-- Begin Jekyll SEO tag v2.8.0 -->
-        <title>{{config.title}} | {{config.description}}</title>
+        <title>{% if page.title %}{{page.title}} - {{config.title}}{% else %}| {{config.description}}{% endif %}</title>
         <meta name="generator" content="Zola v0.17.2">
-        <meta property="og:title" content="{{config.title}}">
+        <meta property="og:title" content="{% if page.title %}{{page.title}}{% else %}{{config.title}}{% endif %}">
         <meta property="og:locale" content="en_US">
-        <meta name="description" content="{{config.description}}">
-        <meta property="og:description" content="{{config.description}}">
-        <link rel="canonical" href="/">
-        <meta property="og:url" content="/">
-        <meta property="og:site_name" content="">
+        <meta name="description" content="{% if page.title %}{{page.title}}{% else %}{{config.description}}{% endif %}">
```

Comment:
I don't think both the title and description should be the page title?
I guess this should maybe still unconditionally be config.description?

Reply:
These are good questions, but I'd like to merge this now to fix the brokenness, then we can come back and refine. Thanks for the careful review!